tests/examples/author_search.feature.example

```diff
@@ -0,0 +1,20 @@
+Feature: Author Search
+
+  Users should be able to search for all or part of an author's name.
+  This should produce a list of sortable results, including variations on possible diacritic letters.
+  - Wildcards allowed except in first letter position
+  - Surname required, forename optional
+
+  Scenario: searching for a surname
+    Given surname is "schroder"
+    When user performs a search
+    Then a list of results is returned
+    And all entries contain the name "schroder" or diacritic variant "Schrøder" or "Schröder" or "Schrœder" (or others)
+
+  Scenario: searching for a partial name
+    Given forename is "W."
+    And surname is "W"
+    When a user performs a search
+    Then a list of results is returned
+    And all entries contain the forename "W."
```

Comment:
Consider: "..at least one author with the forename..."

Reply:
Good point, because there are multiple authors.

Reply:
Question about how the author search functions: if a forename field (or fragment) is given and a surname fragment is also given, do the two fragments get ANDed on a single author name, or on the whole field?

Example: W. W might match "W. Wright" but not "W. Smith, C. Wright"
OR ... W. W matches both "W. Wright" AND "W. Smith, C. Wright"

Reply:
forename AND surname for the same author. So W. W matches "W. Wright" but not "W. Smith, C. Wright"